app/src/main/kotlin/com/simplemobiletools/draw/dialogs/SaveImageDialog.kt

```diff
@@ -104,7 +107,10 @@ class SaveImageDialog(val activity: SimpleActivity, val suggestedExtension: Stri
     }
 
     private fun getInitialFilename(): String {
-        val defaultFilename = "image_${System.currentTimeMillis() / 1000}"
+        // Display date and time in human readable format:
+        val sdf = SimpleDateFormat("yyyy-MM-dd_HH-mm")
+        val defaultFilename = "image_${sdf.format(Date(System.currentTimeMillis()))}"
```

Comment:
A couple notes. Remove the comment please, rename sdf to simpleDateFormat and store the result of sdf.format in some variable, like "formatted" or so. Theres just too many things within that ${} now, it will be $formatted in the new version. Thanks